### DISCUSSION TITLE
EDMC no longer exports ship info to Coriolis

### DISCUSSION THREAD

**Discussion by Jibsman** [EDCD/EDMarketConnector#1876] [Q&A] [answered]
It's been working forever, and now clicking on the ship name no longer opens Coriolis with the ship info.
Output: Ship loadout is checked.
Coriolis: Normal URL = https://coriolis.io/import?data=

I thought there was a place to check or change something like "input ship info to: " with a choice where to send the info.
What am I missing?

Of course, it started working again after a reboot.
Sheesh.

**Comment by denisvvf** [comment]
Same problem here. Is broken in this version.

**Comment by gazorange01** [comment]
Still broken

**Comment by Rixxan** [comment, marked as answer]
Hey there folks! If there are issues, please file a bug report:
https://github.com/EDCD/EDMarketConnector/issues/. When filing, please be
sure to provide all of the requested logs and other information, including
what version of EDMC you are in.

Unfortunately we can't provide support in this forum.
